src/rust/en.mangademon/src/lib.rs

```diff
@@ -56,10 +56,10 @@ fn get_manga_list(filters: Vec<Filter>, page: i32) -> Result<MangaPageResult> {
 				let index = value.get("index").as_int().unwrap_or(0);
 				let ascending = value.get("ascending").as_bool().unwrap_or(false);
 				url.push_str(match (index, ascending) {
-					(0, true) => "&orderby=name%20asc",
-					(0, false) => "&orderby=name%20desc",
-					(1, true) => "&orderby=views%20asc",
-					(1, false) => "&orderby=views%20desc",
+					(0, true) => "&orderby=NAME%20ASC",
+					(0, false) => "&orderby=NAME%20ASC",
+					(1, true) => "&orderby=VIEWS%20ASC",
+					(1, false) => "&orderby=VIEWS%20ASC",
```

Comment:
why did you change the capitalization here? it seems like it works fine either way on the site

Reply:
It is just what the site uses by default for getting new pages, so I thought it would be best to replicate that.

Reply:
You replaced the descending values with ascending btw